res/css/views/rooms/_DecryptionFailureBar.pcss

```diff
@@ -15,70 +15,71 @@ limitations under the License.
 */
 
 .mx_DecryptionFailureBar {
-    --gap: $spacing-8 $spacing-12;
-
-    display: grid;
-    grid-template-areas:
-        "indicator header  header"
-        ".         message message"; /* Empty grid item under grid-area "indicator" */
-    grid-template-columns: min-content auto auto;
-    align-items: start;
+    --gap-row: $spacing-8;
+    --gap-column: $spacing-12;
+    --gap: var(--gap-row) var(--gap-column);
+    --size-icon: 24px;
 
     background-color: $system;
     padding: $spacing-12;
     margin-inline: $spacing-16;
     border-radius: 4px;
-    gap: var(--gap);
-
-    .mx_DecryptionFailureBar_indicator {
-        grid-area: indicator;
 
-        .mx_Spinner,
-        .mx_DecryptionFailureBar_icon {
-            align-self: start;
-        }
+    display: flex;
+    flex-flow: wrap;
```

Comment:
Flex instead of grid to let the buttons wrapped at the bottom. Grid is applied inside `mx_DecryptionFailureBar_start`.